catalyst/callbacks/metrics/sklearn.py

```diff
@@ -0,0 +1,34 @@
+from typing import Callable, Dict, Mapping
+
+import torch
+
+from catalyst.callbacks.metric import FunctionalBatchMetricCallback
+from catalyst.core.runner import IRunner
+from catalyst.metrics import FunctionalBatchMetric
+
+
+class SklearnCallback(FunctionalBatchMetricCallback):
+    """@TODO: Docs."""
+
+    def __init__(
+        self,
+        keys: Mapping[str, str],
+        metric_fn: Callable,
```

Comment:
could we make it [Callable, str]?
and use `metric_fn=sklearn.metrics.__dict__[metric_fn]` in str case? (we also have to check, that metric_fn isin dict.keys)(, btw)

Reply:
done